addons/console/Console.gd

```diff
@@ -164,11 +165,10 @@ func toggle_size() -> void:
 	else:
 		control.anchor_bottom = 1.0
 
-
 func toggle_console() -> void:
 	control.visible = !control.visible
 	if (control.visible):
-		get_tree().paused = true
+		get_tree().paused = pause_enabled
```

Comment:
Rather than setting paused directly here, I think it would be better to check if pause_enabled is set, then set it to true.  It may be that the game was paused for other reasons (ex: in the menu), and using the console with pause_enabled set to false would actually unpause the game.

Reply:
Also need to add the check for unpausing below: get_tree().paused = false